rest_framework_bulk/drf3/mixins.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import print_function, unicode_literals
+import traceback
+from rest_framework import status
+from rest_framework.mixins import CreateModelMixin
+from rest_framework.response import Response
+
+
+__all__ = [
+    'BulkCreateModelMixin',
+    'BulkDestroyModelMixin',
+    'BulkUpdateModelMixin',
+]
+
+
+class BulkCreateModelMixin(CreateModelMixin):
+    """
+    Either create a single or many model instances in bulk by using the
+    Serializer's ``many=True`` ability from Django REST >= 2.2.5.
+
+    .. note::
+        This mixin uses the same method to create model instances
+        as ``CreateModelMixin`` because both non-bulk and bulk
+        requests will use ``POST`` request method.
+    """
+
+    def create(self, request, *args, **kwargs):
+        bulk = isinstance(request.data, list)
+
+        if not bulk:
+            return super(BulkCreateModelMixin, self).create(request, *args, **kwargs)
+
+        else:
+            serializer = self.get_serializer(data=request.data, many=True)
+            serializer.is_valid(raise_exception=True)
+            self.perform_create(serializer)
```

Comment:
@kevin-brown from #24

> `perform_create` is provided by the generics API, and it's most likely the primary override point for users who previously used the old save hooks. I'd recommend creating a new hook, `perform_bulk_create` and embedding logic in there to call `perform_create` if it exists, or fall  back to the old `pre_save` and `post_save` loops if it doesn't.

Did you had any ideas how to do that since `perform_create` simply calls `serializer.save()`?
Since serializer is then responsible for generating multiple objects, I am not sure what we can loop over here.